homeassistant/components/linkplay/const.py

```diff
@@ -4,3 +4,4 @@
 
 DOMAIN = "linkplay"
 PLATFORMS = [Platform.MEDIA_PLAYER]
+CONF_SESSION = "session"
```

Comment:
We don't prefix constants with `CONF_` unless it's a key used for configuration.